layouts/index.html

```diff
@@ -81,25 +81,6 @@ <h2 class="h4">Deploy</h2>
     </div>
   </section>
 
-  <section>
-    <div class="container">
-      <div class="row justify-content-center text-center">
-        <section class="section container-fluid mt-5 pb-3">
-          <h1>Deploying to Airgap with Flux & OCM</h1>
-          <img
-            style="width: 100%; margin: auto; display: block;"
-            class="vidyard-player-embed"
-            src="https://play.vidyard.com/NjNrZF2926RUTSUvkU4MdR.jpg"
-            data-uuid="NjNrZF2926RUTSUvkU4MdR"
-            data-v="3"
-            data-type="inline"
-          />
-          <p>Try it yourself <a href="https://github.com/open-component-model/demo-secure-delivery" target="_blank">here</a>.
-        </section>
-      </div>
-    </div>
-  </section>
-

```

Comment:
this is a major change on the main page... I can't judge if it's ok to be removed... who is our head of communication? ;-)

Reply:
@fabianburth and @morri-son are not so happy with that video on the front page. So, we linked it in the examples.